monero-outreach-docs/translations/es/storyboard/wallet_quickstart_es.md

```diff
@@ -0,0 +1,56 @@
+
+Para máxima seguridad — seguridad incluso si tu dispositivo primario está comprometido por malware — se pueden utilizar monederos hardware especializados. Los monederos hardware son pequeños dispositivos electrónicos que guardan tus claves privadas y se comunican con software en el dispositivo primario usando sólo información pública. Los monederos hardware otorgan seguridad extra por el típico precio de $50-$150.
+
+Así que, con todo esto en mente, considera las siguientes opciones: si deseas ejecutar tu propio nodo completo, una buena elección es el monedero Monero GUI o CLI oficial. El primero provee una interfaz gráfica de usuario, y el segundo una interfaz textual. Ambos pueden ser configurados ya sea para ejecutar un nodo completo o conectarse a un nodo remoto en un modo mejorado de privacidad. Estos pueden ser descargados de [getmonero.org/es/downloads](https://getmonero.org/es/downloads/). Para dispositivos Android, Monerujo, en [monerujo.io](https://www.monerujo.io/), o para iOS, Cake Wallet, en [cakewallet.io](https://cakewallet.io/), son monederos ligeros de protección de claves de visualización. Si quisieras un monedero ligero de compartir claves de visualización, el monedero basado en navegador MyMonero, en [mymonero.com](https://mymonero.com/), se recomienda. The Edge Wallet, en [edge.app](https://edge.app/), es otra sólida opción de monedero ligero de compartir claves de visualización para ambos Android y iOS.
+
+Al escoger entre estos, ten en cuenta que la mayoría tiene una opción extra en la cual — siendo la excepción The Edge Wallet — pueden ser configurados para conectarse ya sea a un servidor remoto de terceros o a un servidor que ejecutes con un nodo. Esto se ilustra en la tabla.
+
+## Monederos Oficiales
+
++ [getmonero.org/es/downloads](https://www.getmonero.org/es/downloads)  
+Si deseas ejecutar tu propio nodo completo, una buena elección es el monedero Monero GUI o CLI oficial. El primero provee una interfaz gráfica de usuario, y el último una interfaz textual.
+
+## Monederos Móviles
+
++ [cakewallet.io](https://cakewallet.io/)  
+Cake Wallet, un monedero ligero de protección de claves-de-visualización, está disponible para dispositivos iOS.
+
++ [monerujo.io](https://www.monerujo.io/)  
+Monerujo, un monedero ligero de protección de claves-de-visualización, está disponible para dispositivos Android.
+
++ [edge.app](https://edge.app/)  
+Edge Wallet está disponible para ambos Android y iOS. Es un monedero ligero de compartir claves de visualización.
+
+## Monederos Web *(Optimizados para Conveniencia)*
+
++ [mymonero.com](https://mymonero.com/)  
+MyMonero es un monedero ligero, basado en navegador. También está disponible como una app de escritorio y app de iOS.
+
+## Monederos Hardware *(Optimizados para Seguridad)*
```

Comment:
The same here:

_(Optimizados para la Seguridad **del usuario**)_

Not sure, maybe I'm confused, just want to hear your thoughts.

Reply:
It sounds a little weird but I can't find another translation, but if we're going to add something, it can be:

+ _Optimizados para mayor seguridad.
+ _Optimizados para una mayor seguridad.

Reply:
Oh yes, this sounds much nicer.

Reply:
- Optimizados para mayor seguridad.